File: app/src/main/java/rus/tutby/adapters/provideradapters/TutPagerAdapter.java
package rus.tutby.adapters.provideradapters;

import android.support.v4.app.FragmentManager;


public class TutPagerAdapter extends ProviderPagerAdapter {

    public TutPagerAdapter(FragmentManager fm, String[] categories, String[] urls) {
        super(fm, categories, urls);
    }

    @Override
    public int getCount() {
        return 10;
    }
}

File: app/src/main/java/rus/tutby/ui/adapters/provideradapters/TutPagerAdapter.kt
package rus.tutby.ui.adapters.provideradapters

import android.support.v4.app.FragmentManager


class TutPagerAdapter(fm: FragmentManager, categories: Array<String>, urls: Array<String>) : ProviderPagerAdapter(fm, categories, urls) {

    override fun getCount(): Int = 10
}
